### ISSUE TITLE
[Feature Request]: make base url configurable

### ISSUE THREAD

**Issue by michael-wirth** [infinispan/infinispan-console#464]
### Feature description

I want to deploy Infinispan service behind a reverse proxy and change the base URL `<base href="/console">`.
It would nicelike to change the base URL by configuration or adapt dynamically depending on the `x-forwarded-prefix` header. e.g. `x-forwarded-prefix: /infinispan`

**Comment by tristantarrant** [comment]
Since the `/console` requires calls to the REST server context which defaults to `/rest`. I assume you will want to move that as well ?

**Comment by michael-wirth** [comment]
@tristantarrant 
Yes, I missed that. I assumed the /rest called would be relative, to the base URL, but I see now they are absolute as well.

**Comment by karesti** [comment]
Currently the console is hardcoded at /console. We should make it configurable to allow loading static resources from custom paths.

**Comment by karesti** [comment]
https://github.com/infinispan/infinispan/issues/14983
